src/types/enrichment/datasource.ts

```diff
@@ -0,0 +1,43 @@
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+import * as t from "io-ts";
+
+const DBParams = t.type({
+  connectionString: NonEmptyString,
+  query: NonEmptyString,
+  whereConditionId: NonEmptyString,
```

Comment:
I'm wondering about this. Where condition should be related to DBType as CosmosDb for example works with `partitionKey` (optional but recommended) and `id`